api/specs/webserver/openapi-projects-with-folder.yaml

```diff
@@ -0,0 +1,122 @@
+paths:
+  /projects-with-folder:
+    get:
+      tags:
+        - project
+      summary: List all projects + folder
+      operationId: list_projects_with_folder
+      parameters:
+        - name: type
+          in: query
+          schema:
+            type: string
+            default: "all"
+            enum: [template, user, all]
+        - name: show_hidden
+          in: query
+          schema:
+            type: boolean
+            default: false
+          description: includes projects marked as hidden in the listing
+        - name: offset
+          in: query
+          schema:
+            type: integer
+            minimum: 0
+            default: 0
+          required: false
+          description: index to the first item to return
+        - name: limit
+          in: query
+          schema:
+            type: integer
+            default: 20
+            minimum: 1
+            maximum: 50
+          required: false
+          description: maximum number of items to return
+      responses:
+        "200":
+          description: list of projects with folder
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProjectWithFolderArrayEnveloped"
+
+        default:
+          $ref: "#/components/responses/DefaultErrorResponse"
+
+  /projects-with-folder/{project_id}:
```

Comment:
so if I get it right, this entrypoint is exactly the same as ```/projects``` but it returns a project that also contains the folder_id it is in?
why does a project need to know in which folder it is located? I mean the folder kinds of "own" the project not the reverse right?

I guess you are using this to list the projects and show the folder when they are first appearing in the list? How will you handle when people start asking to sort first by folder or by projects? or are you already sorting them?
in which case would it not make sense to first list the folders, then get the projects?and as long as you do not open the folder in the UI you do not need to get the projects inside. so why load them at all?